mobly/controllers/android_device_lib/services/logcat.py

```diff
@@ -139,7 +139,9 @@ def create_output_excerpts(self, test_info):
         excerpt_file_path = os.path.join(dest_path, filename)
         with io.open(excerpt_file_path, 'w', encoding='utf-8',
                      errors='replace') as out:
-            while True:
+            # Bad device may unexpected offline during test,
```

Comment:
do you mean "Devices may accidentally go offline during test"?

Reply:
yes, that's what I mean.